components/script/dom/html/htmlformelement.rs

```diff
@@ -1965,9 +1965,11 @@ pub(crate) fn encode_multipart_form_data(
         match entry.value {
             FormDatumValue::String(ref s) => {
                 let content_disposition = format!("form-data; name=\"{}\"", entry.name);
-                let mut bytes =
-                    format!("Content-Disposition: {}\r\n\r\n{}", content_disposition, s)
-                        .into_bytes();
+                let mut bytes = format!(
+                    "Content-Disposition: {}\r\n\r\n{}\r\n",
+                    content_disposition, s
+                )
+                .into_bytes();
```

Comment:
Could be a tiny bit shorter and more readable:

```suggestion
FormDatumValue::String(ref value) => {
let content_disposition = format!("form-data; name=\"{}\"", entry.name);
                let mut bytes = format!(
                    "Content-Disposition: {content_disposition}\r\n\r\n{value}\r\n")
                .into_bytes();
```